plan.go

```diff
@@ -7,134 +7,136 @@ import (
 	ListMeta
-	Values []*Plan `json:"data"`
+	Data []*Plan `json:"data"`
 }
 
 // PlanListParams is the set of parameters that can be used when listing plans.
 // For more details see https://stripe.com/docs/api#list_plans.
 type PlanListParams struct {
 	ListParams   `form:"*"`
-	Created      int64             `form:"created"`
+	Created      *int64            `form:"created"`
 	CreatedRange *RangeQueryParams `form:"created"`
 }
 
 // PlanParams is the set of parameters that can be used when creating or updating a plan.
 // For more details see https://stripe.com/docs/api#create_plan and https://stripe.com/docs/api#update_plan.
 type PlanParams struct {
-	Params         `form:"*"`
-	Amount         uint64                    `form:"amount"`
-	AmountZero     bool                      `form:"amount,zero"`
-	BillingScheme  string                    `form:"billing_scheme"`
-	Currency       Currency                  `form:"currency"`
-	ID             string                    `form:"id"`
-	Interval       PlanInterval              `form:"interval"`
-	IntervalCount  uint64                    `form:"interval_count"`
-	Nickname       string                    `form:"nickname"`
-	Product        *PlanProductParams        `form:"product"`
-	ProductID      *string                   `form:"product"`
-	Tiers          []*PlanTierParams         `form:"tiers,indexed"`
-	TiersMode      string                    `form:"tiers_mode"`
-	TransformUsage *PlanTransformUsageParams `form:"transform_usage"`
-	TrialPeriod    uint64                    `form:"trial_period_days"`
-	UsageType      string                    `form:"usage_type"`
+	Params          `form:"*"`
+	Amount          *int64                    `form:"amount"`
+	BillingScheme   *string                   `form:"billing_scheme"`
+	Currency        *string                   `form:"currency"`
+	ID              *string                   `form:"id"`
+	Interval        *string                   `form:"interval"`
+	IntervalCount   *int64                    `form:"interval_count"`
+	Nickname        *string                   `form:"nickname"`
+	Product         *ProductParams            `form:"product"`
+	ProductID       *string                   `form:"product"`
+	Tiers           []*PlanTierParams         `form:"tiers,indexed"`
+	TiersMode       *string                   `form:"tiers_mode"`
+	TransformUsage  *PlanTransformUsageParams `form:"transform_usage"`
+	TrialPeriodDays *int64                    `form:"trial_period_days"`
+	UsageType       *string                   `form:"usage_type"`
 }
 
 // PlanTier configures tiered pricing
 type PlanTier struct {
-	Amount uint64 `json:"amount"`
-	UpTo   uint64 `json:"up_to"`
+	Amount int64 `json:"amount"`
+	UpTo   int64 `json:"up_to"`
 }
 
 // PlanTransformUsage represents the bucket billing configuration.
 type PlanTransformUsage struct {
-	DivideBy int64  `json:"bucket_size"`
-	Round    string `json:"round"`
+	DivideBy int64                   `json:"bucket_size"`
```

Comment:
What's the detail with this one again? Should we rename to `BucketSize`?

Reply:
damn yeah those were added after I renamed everything so I did not triple check. Also they were just completely broken because the constants had the wrong values lol